src/WebPages/Cart/Cart.jsx

```diff
@@ -0,0 +1,68 @@
+import { HorizontalProductCard } from "../../Components/HorizontalProductCard/HorizontalProductCard";
+import { ProductCard } from "../../Components/ProductCard/ProductCard";
+import { useData } from "../../Contexts/DataContext";
+import "./Cart.css";
+const Cart = () => {
+  const { state } = useData();
+  const CartData = state.productdata.filter((ele) => ele.inCart === true);
+
+  const priceOfItems = CartData.reduce(
+    (prev, curr) => prev + Number(curr.price) * Number(curr.qty),
+    0
+  );
+  return (
+    <>
+      {CartData.length ? (
+        <>
+          <div class="cart-heading">
+            <h1 class="small-heading-text flex-justify-center margin-1rem">
+              My cart({state.cart.length})
+            </h1>
+          </div>
+
+          <div class="cart-container margin-1rem">
+            <div className="cart-cards">
+              {CartData.map((ele) => {
+                return <HorizontalProductCard key={ele._id} product={ele} />;
+              })}
+            </div>
+
+            <div class="price-card">
+              <div class="card-heading">
+                <h2>Price Details</h2>
+              </div>
+
+              <div class="price-calculation">
+                <div class="price-components display-flex">
+                  <p>Price({CartData.length})</p>
+                  <p> {priceOfItems}</p>
+                </div>
+                <div class="price-components display-flex">
+                  <p>Discount</p>
+                  <p>-500</p>
+                </div>
+                <div class="price-components display-flex">
+                  <p>Delivery</p>
+                  <p>+299</p>
```

Comment:
Nice structured code, just a suggestion you can keep the delivery fee in a `const` variable and then reuse it rather than hard coding every time